sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -104,7 +106,18 @@ object AkkaserverlessPlugin extends AutoPlugin {
     Test / managedSources :=
       (Test / managedSources).value.filter(s => !isIn(s, (Test / temporaryUnmanagedDirectory).value)),
     Test / unmanagedSources :=
-      (Test / generateUnmanaged).value ++ (Test / unmanagedSources).value)
+      (Test / generateUnmanaged).value ++ (Test / unmanagedSources).value,
+    onlyUnitTest := {
+      sys.props.get("onlyUnitTest") match {
+        case Some("true") => true
+        case _            => false
+      }
```

Comment:
that could be simplified to 
```
onlyUnitTest := Boolean.getBoolean("onlyUnitTest")
```

(yeah, that is plain old Java)

Reply:
:) niiice! 

About the docs, will do. Actually, I was about to add it but realized there's no unique place where there's a reference to this, and also that it doesn't appear on "Implementing Event Sourced Entities" nor in "Implementing Actions". I'll add it accordingly.